src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -475,3 +484,44 @@ bool java_bytecode_languaget::to_expr(
 java_bytecode_languaget::~java_bytecode_languaget()
 {
 }
+
+/// Activate the generic instantiation code.
+/// \param message_handler
+/// \param symbol_table The symbol table so far.
+void
+java_bytecode_languaget::handle_generics(
+  message_handlert &message_handler,
+  symbol_tablet &symbol_table)
+{
+  generate_java_generic_typet instantiate_generic_type(message_handler);
+  // check out the symbols in the symbol table at this point to see if we
+  // have a a generic type in.
+  for(const auto &symbol : symbol_table.symbols)
+  {
+    if(symbol.second.type.id()==ID_struct)
```

Comment:
I suspect we want to include not just component members but also parameters to functions - but let us leave that for now